app/helpers/ocl_tools/components_helper.rb

```diff
@@ -73,8 +73,12 @@ def header(*args, &blk)
       render HeaderComponent.new(*args, &blk)
     end
 
-    def table(*args, **kwargs, &blk)
-      render TableComponent.new(*args, **kwargs, &blk)
+    def search(*args, &blk)
+      render SearchComponent.new(*args, &blk)
+    end
+
+    def table(*args, row_url: nil, small: false, &blk)
+      render TableComponent.new(*args, row_url: row_url, small: small, &blk)
```

Comment:
What was behind this change? I'm now getting
```
unknown keyword: :show_header
```
in various places where the component is used.

Is it ok to switch it back to `**kwargs`?